libs/UIHelpers/transfer_function_widget.cpp

```diff
@@ -37,547 +15,525 @@
 #include "stb_image.h"
 #include "stb_image_resize.h"
 
-template <typename T>
-T clamp(T x, T min, T max)
+static const float alpha_points_min_distance = 0.006f;
+
+namespace tfnw
 {
-  if (x < min)
+  template <typename T>
+  inline T clamp(T x, T min, T max)
   {
-    return min;
+    if (x < min)
+    {
+      return min;
+    }
+    if (x > max)
+    {
+      return max;
+    }
+    return x;
```

Comment:
Are you adding any extra functionality here? It's a little hard to tell in GitHub, just want to make sure you're not merging any changes you don't mean to

Reply:
Yes. Kind of new functionality. Updating imgui also implies upgrading the code in the transfer function widgets.